python/GafferUI/SpreadsheetUI/_CellPlugValueWidget.py

```diff
@@ -176,6 +176,8 @@ def walk( widget ) :
 			if isinstance( widget, GafferUI.NumericPlugValueWidget ) :
 				widget._qtWidget().setFixedWidth( cls.__numericFieldWidth )
 				widget._qtWidget().layout().setSizeConstraint( QtWidgets.QLayout.SetNoConstraint )
+			elif isinstance( widget, GafferUI.StringPlugValueWidget ) :
+				widget.textWidget().setPreferredCharacterWidth( 60 )
```

Comment:
> What might be even better is to match the widget width to the column width

Yep. We're already doing that for VectorDataPlugValueWidgets at L192 below, so this might be as simple as updating the `isinstance()` call to include StringPlugValueWidget.

Reply:
That's great, I moved the `StringPlugValueWidget` sizing down into that block and squashed it down into https://github.com/GafferHQ/gaffer/pull/5484/commits/5dd723c17d254cfad3dbbc631ccbcd43d61a1f29